src/runmodes.c

```diff
@@ -795,7 +795,7 @@ void RunModeInitializeOutputs(void)
             }
         }
         if (count == 0) {
-            FatalErrorOnInit(SC_ERR_INVALID_ARGUMENT,
+            SCLogWarning(SC_ERR_INVALID_ARGUMENT,
```

Comment:
can we add the dns-log check above where eve/libjansson is checked as well?

Reply:
You mean like a specific check for the dns-log type? Rather than a generic warning for unknown? Where unknown may be an old unsupported?